wp-content/civi-extensions/civiglific/README.md

```
@@ -1,12 +1,77 @@
+2. **Sending receipts and messages to contributors through WhatsApp**
+
+### Setup 1: Migrate CiviCRM Group Contacts to Glific Groups
+
+1. **Add configuration details in your `settings.php` file** to connect Glific with CiviCRM:
+   ```php
+   define('CIVICRM_GLIFIC_PHONE', 'XXXXXXXXXX');
+   define('CIVICRM_GLIFIC_PASSWORD', 'ABC');
+   define('CIVICRM_GLIFIC_API_BASE_URL', 'https://api.abs.com');
+   ```
+2. **Add a navigation menu entry** in CiviCRM to access the Group Mapping screen. You can use the following URL:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)